crates/node/Cargo.toml

```diff
@@ -54,7 +55,7 @@ sp-trie = { workspace = true, features = ["default"] }
 # These dependencies are used for the node template's RPCs
 jsonrpsee = { workspace = true, features = ["server"] }
 
-# Substrate primtiives dependencies
+# Substrate primtives dependencies
```

Comment:
typo: `primitives`

Reply:
Hahah, fixed one typo and got another in... XD